### PR TITLE
Editor Window & Documentation.txt

### PR DESCRIPTION
**Pull request by ah1053**
1.Replaced dummy.txt in the Documentation Directory with Documentation.txt
2.Updated MobiledgeX Unity Editor links
3.Full description of Enhacement.dll [here](https://github.com/mobiledgex/unity-sdk-tracker/pull/2)
4.EdgeMultiplay Update  [here](https://github.com/mobiledgex/edge-multiplay-unity-client/pull/2)

### CODE REVIEW

File: Editor/MobiledgeXEditorWindow.cs [hunk 1]
```diff
@@ -109,32 +109,33 @@ public static void ImportEdgeMultiplayExample()
...
-                         Directory.Delete(Path.Combine("Assets", "Plugins/MobiledgeX"), true);
-                         File.Delete(Path.Combine("Assets", "Plugins/MobiledgeX")+".meta");
-                    }
-                    AssetDatabase.Refresh();
-                    Client.Remove("com.mobiledgex.sdk");
-                    EditorApplication.Exit(0);
+                    Directory.Delete(Path.Combine("Assets", "Plugins/MobiledgeX"), true);
+                    File.Delete(Path.Combine("Assets", "Plugins/MobiledgeX") + ".meta");
                 }
+                EditorPrefs.DeleteKey("mobiledegx-user");
```
Comment (franklin-huang-mobiledgex) [inline Editor/MobiledgeXEditorWindow.cs:134]: typo

File: Editor/MobiledgeXEditorWindow.cs [hunk 2]
```diff
@@ -485,6 +478,13 @@ void SetUpLocationSettings()
             PlayerSettings.iOS.locationUsageDescription = "Geo-Location is used by MobiledgeX SDK to locate the closest edge cloudlet server and (where supported) for carrier enhanced Verify Location services.";
         }
 
+        static string getId()
+        {
+            string id = EditorPrefs.GetString("mobiledegx-user", Guid.NewGuid().ToString());
```
Comment (franklin-huang-mobiledgex) [inline Editor/MobiledgeXEditorWindow.cs:483]: typo (mobiledegx -> mobiledgex)

File: Editor/MobiledgeXEditorWindow.cs [hunk 3]
```diff
@@ -485,6 +478,13 @@ void SetUpLocationSettings()
...
+            EditorPrefs.SetString("mobiledegx-user", id);
```
Comment (franklin-huang-mobiledgex) [inline Editor/MobiledgeXEditorWindow.cs:484]: same as above

File: Editor/MobiledgeXEditorWindow.cs [hunk 4]
```diff
@@ -493,5 +493,5 @@ void SetUpLocationSettings()
 public class PackageDetails
 {
     public string version;
-    
+
```
Comment (franklin-huang-mobiledgex) [inline Editor/MobiledgeXEditorWindow.cs:496]: nit extra line

**Review by franklin-huang-mobiledgex — APPROVED** [review]
Looks good to me other than a couple of typos

**Review by brucearmstrong — APPROVED** [review]
Good updates.

File: Runtime/Scripts/ExampleUDP.cs [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+    IEnumerator Start()
+    {
+        yield return StartCoroutine(MobiledgeX.LocationService.EnsureLocation());
+        GetEdgeConnection();
+    }
+
+    async void GetEdgeConnection()
+    {
+        mxi = new MobiledgeXIntegration();
```
Comment (lgarner) [inline Runtime/Scripts/ExampleUDP.cs:24]: Is there a warning message anywhere it isn't actually edge in some cases?

**Review by lgarner — APPROVED** [review]
Ok. What's getID for?

**Review by lgarner — COMMENTED** [review]
👍

**Pull request merged**